lib/modules/manager/flux/readme.md

```diff
@@ -20,6 +20,14 @@ In addition, for the `flux` manager to properly link `HelmRelease` and `HelmRepo
 
 Namespaces will not be inferred from the context (e.g. from the parent `Kustomization`).
 
+As for `HelmRelease` resources that sourced from `GitRepository`, those will be updated by updaing the `GitRepository` resource.
```

Comment:
```suggestion
Renovate updates `HelmRelease` resources coming from `GitRepository` by updating the `GitRepository` resource.
```

We're using `resources` and `sourced` too much. I've tried to rewrite this sentence. I'm still not happy with it, but it's better than before.

Reply:
Thank you @HonkingGoose , Your sentences are much clearer I like them and happy with them.